examples/marketing-model/README.md

```diff
@@ -0,0 +1,7 @@
+# Build a machine learning marketing model for banking using Google Cloud Platform and Python
+
+This notebook shows you how to build a marketing model for banking using Google Cloud Platform (GCP). Many financial institutions use traditional on-premise methods and tooling to build models for marketing. This notebook takes you through the steps to build a Machine Learning model using open source and Google Cloud Platform.
```

Comment:
Can we add some more description here as to what the model does. I don't feel someone will know much about this repo folder until they open the notebook and read through it.